main/webapp/modules/core/scripts/dialogs/common-transform-dialog.html

```diff
@@ -0,0 +1,12 @@
+<div class="dialog-frame" style="width: 400px;">
+    <div class="dialog-header" bind="dialogHeader"></div>
+    <div class="dialog-body" bind="dialogBody"><div class="grid-layout grid-layout-for-ui layout-normal layout-full"> 
+    <td width="100%"><div id="checkedColumn" class="column-reordering-dialog-column-container" bind="columnContainer"></div></td>
+    <div  class="all-column-transform-dialog-container dialog-footer" bind="dialogFooter" style="font-size: small;" >
```

Comment:
I think we should avoid explicitly setting `font-size: small` here, but rather rely on the existing CSS classes. It seems to me that the two divs opened above (`<div class="dialog-body" bind="dialogBody"><div class="grid-layout grid-layout-for-ui layout-normal layout-full">`) are not closed anywhere, which might be the cause of the styling issue.

Reply:
I changed it ,please see once